hello.py

```diff
@@ -103,14 +105,40 @@ def tmp_copy(src):
     return dst
 
 
+def get_current_time():
+    return datetime.now()
+
+
+def delta_time(t_end, t_start):
+    delta = t_end - t_start
+    return delta.total_seconds(), delta.microseconds
+
+
+@contextmanager
+def timer(slogan):
```

Comment:
Shall we rename `slogan` to `cmd`?
slogan is a indicator telling what is being done in the context manager

Reply:
Done.